index.ts

```diff
@@ -63,15 +57,16 @@ export const typescriptPaths = ({
 				return null;
 			}
 
-			const targetFileName = join(
-				outDir,
+			// TODO: Do we need outDir as "resolvedFileName" is already correct absolute path
```

Comment:
I think `resolvedFileName` is only the correct absolute path with Rollup 3 but not Rollup 2 or so.